### PR TITLE
UAF-2492:Procurement Cardholder Maintenance Step 1 (Add New Fields to Default Table)

### PR DESCRIPTION
**Pull request by nataliac06**
Changed ProcurementCardDefault which will replace ProcurementCardDetails in KFS 3.0 and added additional fields to the FP_PRCRMNT_CARD_DEFAULTS, fp-ojb repo and data dictionary in the arizona.edu package.
Modified the lookup to add the fields from the spec.
Added a couple of "hard-coded" card status and card cancel status value finders to help populate the drop downs in the lookup.

### CODE REVIEW

File: kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/ProcurementCardDefault.xml [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+        </property>
+      </bean>
+    </property>
+  </bean>
+
+	<bean id="ProcurementCardDefault-creditCardLastFour" parent="ProcurementCardDefault-creditCardLastFour-parentBean" />
+	<bean id="ProcurementCardDefault-creditCardLastFour-parentBean" abstract="true" parent="AttributeDefinition">
+		<property name="name" value="creditCardLastFour" />
+		<property name="forceUppercase" value="false" />
+		<property name="label" value="Card Number Last Four" />
```
Comment (ghost) [inline kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/ProcurementCardDefault.xml]: KFS 3.0 label for this field is "Procurement Card Number Last Four". You might want to check with Misty on whether this label change is OK.

**Review by ghost — COMMENTED** [review]
Except for the one label change I noticed, looks good.

File: kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/ProcurementCardDefault.xml (hunk 1 above)
Comment (nataliac06) [reply kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/ProcurementCardDefault.xml]: I'll change the label in the DD, I probably have overlooked it.

**Comment by ghost** [comment]
Looks good!

**Pull request merged**